core/README.md

```diff
@@ -3,58 +3,51 @@
 
-WARNING: This repository contains a version 1, “beta release” of the IOF Core Ontology. As such, the contents may undergo some change as we resolve outstanding issues and drive to a first stable release. 
+# Contents
 
-# Contents of this Repository
+1. core.rdf – an OWL implementation of the IOF Core Ontology, inclusive of natural language definitions, formalizations in first-order logic, examplyes, and comments, serialized in rdf/xml.
+1. meta – folder containing IOF-specific annotation properties used to define and formalize ontology terms, also serialized in rdf/xml, and imported by the core ontology. 
 
-1. core.rdf – an OWL implementation of the IOF Core Ontology, inclusive of natural language definitions, formalizations in first-order logic, examples, and comments, serialized in rdf/xml.
-1. meta – a directory containing IOF-specific annotation properties used to define and formalize ontology terms, also serialized in rdf/xml, and imported by the core ontology.
-1. LICENSE – a file containing details of the OAGi copyright statement, licensing and permissions granted for this ontology.
-1. Catalog-v001.xml – a tooling file for rapid loading of the ontology in the IOF-recommended tool for viewing and editing ontologies.
+    * AnnotationVocabulary.rdf -- this is the released version of the IOF Annotation Vocabulary, dated 2022-09-29
+1. commons-to-core-mapping – folder containing selected mid-level ontologies from the OMG Commons Library of Ontologies, addressing IOF needs in the topic area of identifiers and designators, as well as mapping files to map imported terms to IOF/BFO and align annotation property usage between IOF and OMG.  
+
+	* Commons Folder -- The various OMG Commons Ontology files adopted by Core in this release, including Collections.rdf, Designators.rdf, TextDatatype.rdf and Identifiers.rdf
+	* meta folder
+	    * MappingAnnotationVocabularyToCommons.rdf -- Maps the somewhat different OMG annotation vocabulary to the IOF annotation vocabulary
+	* MappingCommonsToIOF.rdf -- maps terms from the adopted OMG Commons ontologies to BFO and/or to IOF
+1. Catalog-v001.xml – a tooling file for rapid loading of the ontology's files in the Protégé ontology development tool.
 
 # Installing / Getting started
 
 The minimal setup you need short of reading raw XML is a suitable ontology viewer or editing tool, installed on your client machine. IOF recommends a desktop version of the open-source tool [Protégé](https://protege.stanford.edu/), but other open source and commercial tools are known to work as well. The repository for this ontology includes the necessary files for opening the ontology in Protégé without warnings and errors. The same is not guaranteed for other tools. 
```

Comment:
```suggestion
The minimal setup you need short of reading raw XML is a suitable ontology viewer or editing tool, installed on your client machine. IOF recommends a desktop version of the open-source tool [Protégé](https://protege.stanford.edu/), but other open source and commercial tools are known to work as well. The repository for this ontology includes the necessary files for opening the ontology in Protégé without warnings and errors. The same is not guaranteed for other tools. 

To use IOF Core without the need to interoperate with OMG Common ontologies, simply import Core.rdf into your ontology. 

To use IOF Core with OMG Common ontologies, import MappingCommonsToIOF.rdf into your ontology. This indirectly import IOF Core and OMG Common ontologies into your ontology.
```

Reply:
I made these changes in a new version of the file (vs using the "commit suggestion" feature of GitHub. Have another look if you wish.